### ISSUE TITLE
Save video with faces detected marks

### ISSUE THREAD

**Issue by ghost** [EzequielAdrianM/Camera2Vision#7]
Can be possible to save the face detected video with faces marks?

**Comment by EzequielAdrianM** [comment]
Yes, it is possible. But it is not a feature that the Android Camera can do by default. You will need to record a normal video and add the landmarks later on with ffmpeg.

**Comment by ghost** [comment]
Thank you very much, could you redirect me some tutorial on this?

**Comment by ark7** [comment]
yea some tutorial will be great, maybe for next dev, is it possible to record the video base on onUpdateFace signaling, for instance, as long as face detected, the videorecorder will activated, when the face out of frame, the recording just stop.
